### ISSUE TITLE
How to send list of dict in params?

### ISSUE THREAD

**Issue by GinTR1k** [armills/jsonrpc-async#3]
Hello!

How to send list of dict in params?  
```
params: [{
  "from": "...",
  "to": "..."
}]
```

**Comment by emlove** [comment]
You should be able to spread the list as args like this:
```python3
params = [{
  "from": "...",
  "to": "..."
}]
await server.foo(*params)
```

**Comment by emlove** [comment]
OK, FYI this is probably the same issue that was happening here: https://github.com/emlove/jsonrpc-websocket/issues/9 , and I've released a new version 2.0.0 that should fix this issue. You should be able to just pass the dict as the first positional argument and it will work as expected.
